frontend/js/src/explore/fresh-releases/components/ReleaseFilters.tsx

```diff
@@ -32,6 +42,7 @@ type ReleaseFiltersProps = {
   setShowFutureReleases: React.Dispatch<React.SetStateAction<boolean>>;
   releaseCardGridRef: React.RefObject<HTMLDivElement>;
   pageType: string;
+  currentUser: ListenBrainzUser;
```

Comment:
This should be acquired directly using a React Context hook that we have set up for it, instead of being passed by the parent as a prop.

As an example of that:
https://github.com/metabrainz/listenbrainz-server/blob/f346f6aa994acfbcc616c233f8a581ef303ae6a8/frontend/js/src/components/Navbar.tsx#L8